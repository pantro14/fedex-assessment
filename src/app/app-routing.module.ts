import { NgModule } from '@angular/core';
import { RouterModule, Routes } from '@angular/router';

const routes: Routes = [
  { path: '', pathMatch: 'full', redirectTo: 'sign-up' },
  {
    path: 'sign-up',
    loadComponent: () =>
      import('./components/sign-up/sign-up.component').then(
        (m) => m.SignUpComponent
      ),
  },
];

@NgModule({
  imports: [RouterModule.forRoot(routes)],
  exports: [RouterModule],
})
export class AppRoutingModule {}
