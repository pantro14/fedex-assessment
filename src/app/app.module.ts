import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';

import { AppRoutingModule } from './app-routing.module';
import { AppComponent } from './app.component';
import { NavBarComponent } from './common/nav-bar/nav-bar.component';
import {HttpClientModule} from '@angular/common/http';

@NgModule({
  declarations: [AppComponent],
  imports: [BrowserModule, AppRoutingModule, NavBarComponent, HttpClientModule],
  providers: [],
  bootstrap: [AppComponent],
})
export class AppModule {}
