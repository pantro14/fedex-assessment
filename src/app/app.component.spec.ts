import { AppComponent } from './app.component';

describe('AppComponent', () => {
  let appComp: AppComponent;
  beforeEach(() => {
    appComp = new AppComponent();
  });

  fit('should create the app', () => {
    expect(appComp).toBeDefined();
  });
});
