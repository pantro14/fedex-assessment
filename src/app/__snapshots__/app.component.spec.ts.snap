// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent should match the snapshot 1`] = `
<app-root>
  <section
    class="quicksand"
  >
    <app-nav-bar>
      <header
        class="navbar sticky top-0 py-5 z-50 flex justify-center bg-fedex-blue"
      >
        <img
          alt="fedex"
          height="150"
          src="assets/fedex-logo.png"
          width="150"
        />
      </header>
    </app-nav-bar>
    <router-outlet />
  </section>
</app-root>
`;
